CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ version directory, and then changes are introduced.
 - Changed some remaining images to be pulled from Giant Swarm's registry
 - Updated Alpine sidecar for Ingress Controller to version 3.7
 - Fixed mkfs.xfs for containerized kubelet.
+- Updated hyperkube to version 1.10.3.
```

Comment:
Do we finish a change with a `.` or not? Half of them have it and half of them don't.

Reply:
Good catch. IMO they should end with a `.`. There were a couple missing, now done.

Reply:
Thanks <3